blark/iec.lark

```diff
@@ -791,4 +793,4 @@ repeat_statement: "REPEAT"i statement_list "UNTIL"i expression "END_REPEAT"i ";"
 
 exit_statement.1: "EXIT"i ";"*
```

Comment:
I think `EXIT` will need this modification, too

Reply:
```suggestion
exit_statement.1: "EXIT"i ";"+
```

Reply:
Aha! Totally! And an easy enough fix to make, too!